evap/settings.py

```diff
@@ -349,17 +350,17 @@ class ManifestStaticFilesStorageWithJsReplacement(ManifestStaticFilesStorage):
 
 # Additional locations of static files
 STATICFILES_DIRS = [
-    os.path.join(BASE_DIR, "static"),
+    MODULE / "static",
 ]
 
 # Absolute path to the directory static files should be collected to.
-STATIC_ROOT = os.path.join(BASE_DIR, "static_collected")
+STATIC_ROOT = DATADIR / "static_collected"
```

Comment:
Since we are already setting the variable here, it might be counterintuitive if someones does
```python
from evap.settings import *

DATADIR = Path("...")
```
Do we care (now)?

Reply:
We can prefix with `EVAP_` if you anticipate problems, but best practice is to not start-import, and you'll always have the chance to use another name locally for an imported symbol, so for me this is fine as it currently is.

Reply:
I didn't mean that it would be a problem of someone mistakenly overriding our settings, but intentionally trying to do so, like in my proposed settings here: https://github.com/e-valuation/evap-deployment/blob/7f30a50bd2dbbfe157a85fb33e6ed937d6cb3266/productionsettings.template.py

I think it would be nice if it would work this way, but it doesn't because the settings like `MEDIA_ROOT` etc. are already set using the old value of `DATADIR`. Anyways, I suppose it's fair to not care right now

Reply:
Ah, yeah. Yeah I'd say you just can't override `DATADIR`, you have to overide the derived values - not a big deal.